ntbs-service/wwwroot/source/helpers.ts

```diff
@@ -1,6 +1,7 @@
 var getHeaders = function() {
+    let token = (<HTMLInputElement>document.querySelector('[name="__RequestVerificationToken"]'));
     return {
-        "RequestVerificationToken": (<HTMLInputElement>document.querySelector('[name="__RequestVerificationToken"]')).value
+        "RequestVerificationToken": token == null ? token : token.value

```

Comment:
`token == null ? null : token.value` seems to be the same but less confusing?
If there's a chance that the token is not available, do we need to make sure the `Heartbeat` POST does not need this token to be present?